src/cobra/core/reaction.py

```diff
@@ -76,6 +76,7 @@ def __init__(
     ):
         Object.__init__(self, id, name)
         self._gene_reaction_rule = ""
```

Comment:
This was added because there are some bits of code using it as opposed to "reaction.gene_reaction_rule". 
If I remove it, should I correct the other bits of code?

Reply:
The ones in `modify.py` right? In this case I think it's okay to change it to `rxn.gene_reaction_rule` in there. Those are not part of the core API.

Reply:
Done.